package.json

```diff
@@ -5,7 +5,7 @@
   },
   "scripts": {
     "local-lerna": "lerna",
-    "test": "nyc --require @std/esm --reporter=lcov --reporter=text mocha",
+    "test": "nyc --require @std/esm --require ./infra/testing/mock-sw-env --reporter=lcov --reporter=text mocha",
```

Comment:
Optional: we're already using a `nyc` section in `package.json` for some config, and we could move the `--require` and `--reporter` config to there as well, instead of explicitly listing them as CLI arguments.

Reply:
Moving report to nyc, leaving requires as is since we can use it for mocha, so useful seeing it where it's used.